theme/imagecaption/theme.scss

```diff
@@ -2,27 +2,14 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 @import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_colors.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_shadow.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_states.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_spacing.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_fonts.scss';
 
 .ck-widget.image {
```

Comment:
It'd be better if these styles had nothing to do with the widget classes. The widgets theme use widget classes (and hence applies to the editor only) and the image theme uses image classes.

Actually, this is a bigger topic. For the widgets we're defining styles which will apply in a context of the editor cause we use editor classes like `.ck-widget`. For the content (like `figure.image`) we want to provide generic content styles like in the CKEditor 4's [`contents.css`](https://github.com/ckeditor/ckeditor-dev/blob/master/contents.css). So, they need to be overrideable but still so specific to apply to editor contents only, not the whole page.

The only idea I've got now is to prefix all of them with `.ck-editor__editable`. WDYT, @oleq?